src/abci/transaction/hash.rs

```diff
@@ -11,7 +11,7 @@ use subtle::{self, ConstantTimeEq};
 use subtle_encoding::hex;
 
 /// Size of a transaction hash in bytes
-pub const LENGTH: usize = 20;
+pub const LENGTH: usize = 32;
```

Comment:
I introduced HASH_SIZE in https://github.com/interchainio/tendermint-rs/blob/master/src/merkle.rs#L6 ... we'll need to de-duplicate eventually but for now it's fine ....